docs/field-types/array.mdx

```diff
@@ -0,0 +1,57 @@
+---
+id: array
+title: Array
+slug: /field-types/array
+---
+
+import Video from "../../src/components/Video.js";
+import videoUrl from "./assets/array-field.mp4";
+import array1 from "./assets/array1.png";
+import array2 from "./assets/array2.png";
+import arrayExample from "./assets/array-example.png";
+
+The Array field is a field that allows you to store an **array of values**. Rowy Array mimics a **JSON Array** which is a data structure that stores an ordered collection of values in a format known as *JavaScript Object Notation (JSON)*.
+
+<Video url={videoUrl} playing loop muted playsinline controls />
+
+## Supported Values
+
+A JSON Array is a collection of values, which **can be of any type** including `Number`, `Short Text`, another `JSON Object`, `Toggle`, `GeoPoint`, `Date & Time`, or even `Reference`. The values in a JSON Array are **separated by commas** and **enclosed in square brackets**. 
+
+```json
+[
+  "string", // a short text
+  1, // a number
+  true, // a boolean
+  {
+    "key": "value" // an object
+  },
+  [
+    "array" // an array
+  ]
```

Comment:
array inside an array is not possible, you can remove this

Reply:
@iamanishroy, fixed this in PR https://github.com/rowyio/docs/pull/57.